coalib/settings/FunctionMetadata.py

```diff
@@ -186,11 +185,10 @@ def from_function(cls, func, omit=frozenset()):
                     argspec.annotations.get(arg, None))
             else:
                 optional_params[arg] = (
-                    doc_comment.param_dict.get(arg, cls.str_nodesc) + ' (' +
-                    cls.str_optional.format(
-                        defaults[i-num_non_defaults]) + ')',
+                    doc_comment.param_dict.get(arg, cls.str_nodesc),
                     argspec.annotations.get(arg, None),
-                    defaults[i-num_non_defaults])
+                    defaults[i-num_non_defaults],
+                    'Optional')
```

Comment:
Or we can store just the default value alone - what I meant was, we could construct the "Optional, defaults to..." string whenever we want.